nix-tools-iohk-module.nix

```diff
@@ -26,69 +27,85 @@ let
     # run on the target host.
     inherit (config.hsPkgs.remote-iserv.components.exes) remote-iserv;
     # we need to use openssl.bin here, because the .dll's are in the .bin expression.
-    extra-test-libs = [ pkgs.rocksdb pkgs.openssl.bin pkgs.libffi ];
-  } // {
-    # we can perform testing of cross compiled test-suites by using wine.
-    # Therfore let's enable doCrossCheck here!
-    doCrossCheck = pkgs.stdenv.hostPlatform.isWindows;
   };
-in {
-  packages = {
-    # This needs true, otherwise we miss most of the interesting
-    # modules.
-    ghci.flags.ghci = true;
-    # I hope we can apply this globally.
-    ghc.flags.ghci = true;
+  windowsArgs = withTHArgs // { extra-test-libs = [ pkgs.rocksdb pkgs.openssl.bin pkgs.libffi ]; };
+  linuxArgs = withTHArgs // { extra-test-libs = []; };
+in lib.mkMerge [
```

Comment:
How is `mkMerge` different from `//`.  I think it's worth adding a comment here, such that the next person don't make the same mistake?